lib/travis/api/v3/queries/trials.rb

```diff
@@ -4,5 +4,15 @@ def all(user_id)
       client = BillingClient.new(user_id)
       client.trials
     end
+
+    def create(user_id)
+      client = BillingClient.new(user_id)
+      login = client.create_trial(params['type'], params['owner'])
```

Comment:
@piotr-travisci, do we check somewhere that user who made the request belongs to this organization and has permissions to do this?

Reply:
Of course. We are checking it in billing-v2 app.